pkg/cluster/healthcheck.go

```diff
@@ -83,13 +83,13 @@ func (u *simpleUpgrader) HealthCheck(ctx context.Context, cs *api.OpenShiftManag
 	}
 
 	u.log.Info("checking developer console health")
-	err = u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs.Properties.FQDN, cs.Config.Certificates.OpenShiftConsole.Cert), "https://"+cs.Properties.PublicHostname+"/console/", time.Second)
+	cert := cs.Config.Certificates.OpenShiftConsole.Cert
+	err = u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs.Properties.FQDN, cert[len(cert)-1]), "https://"+cs.Properties.PublicHostname+"/console/", time.Second)
 	if err != nil {
 		return err
 	}
-
 	u.log.Info("checking admin console health")
-	return u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs.Properties.RouterProfiles[0].FQDN, cs.Config.Certificates.Router.Cert), "https://console."+cs.Properties.RouterProfiles[0].PublicSubdomain+"/", time.Second)
+	return u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs.Properties.RouterProfiles[0].FQDN, cert[len(cert)-1]), "https://console."+cs.Properties.RouterProfiles[0].PublicSubdomain+"/", time.Second)
```

Comment:
Strictly you should be using the Router CA cert here (they're probably the same, but)